### DISCUSSION TITLE
How to get automatic deployment with pre- v0.3.5

### DISCUSSION THREAD

**Discussion by Emadmasroor** [alshedivat/al-folio#368] [Q&A]
I forked your repository to build my [website](https://github.com/Emadmasroor/emadmasroor.github.io) before version 0.3.5, so I don't have automatic deployment to the `gh-pages` branch. When I make changes to `master`, `gh-pages` does not update.

How do I get `gh-pages` to update automatically? Is there a good way for me to update to v0.3.5 of your template? I would like to be able to update without having to start all my content from scratch..

**Comment by alshedivat** [comment]
good question. I think the easiest way would be to copy [`.github/workflow/` folder](https://github.com/alshedivat/al-folio/tree/master/.github/workflows) to your repo, then follow [the deployment instructions](https://github.com/alshedivat/al-folio#deployment) related to setting up github actions (see "For personal and organization webpages" section).
